src/main/java/com/clevertap/apns/Notification.java

```diff
@@ -45,20 +45,29 @@
     private final String token;
     private final String topic;
     private final String collapseId;
+    private final Long expiration;
```

Comment:
Let's make this into a primitive with the default as -1? So only include the header if the value is greater than -1 (as 0 is a valid value).